Documentation/concepts/networking/fragmentation.rst

```diff
@@ -0,0 +1,22 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _concepts_fragmentation:
+
+IP fragment handling (beta)
+===========================
+
+By default, Cilium configures the eBPF datapath to perform IP fragment tracking
+to allow protocols that do not support segmentation (such as UDP) to
```

Comment:
maybe
```suggestion
to allow protocols that do not support fragmentation (such as UDP) to
```

Reply:
This one was deliberate, although they're synonyms I've tended to see `segmentation` used to talk about breaking up messages into datagrams at layer 4 while people seem to use `fragmentation` to refer to layer 3 message partitioning. (Reviewers, I'd be interested to hear if you disagree)